### PR TITLE
Display Payara Server Community (web profile) as a version instead of a separate product

### LINKED ISSUE #1592: Compatible Software Product Request for Payara Server Community (Web Profile) 6.2022.1 For Web Profile 10 and Core Profile 10
To include your Compatible Software Product on the Jakarta EE official list of Compatible Software Products, please provide the following information:

**The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting**
https://accounts.eclipse.org/users/jameshillyard

**A product name and version**
Payara Server Community 6.2022.1 (Web Profile)

**The Profile (Full Platform, Web Profile, Core Profile) that the Compatible Product is submitted to be listed under**
Web Profile 10 and Core Profile 10

**A ZIP file with a product logo with a transparent background in SVG format (optional)**
https://github.com/jakartaee/jakarta.ee/files/9605791/community-logo-on-white-transparent-RGB.zip

**The Participant, Enterprise or Strategic Member name supporting the Compatible Software Product**
Payara Services Ltd

**The URL to the approved certification issue demonstrating compatibility as defined in the Jakarta EE TCK Process**
Full Platform: https://github.com/eclipse-ee4j/jakartaee-platform/issues/578
Core Profile: https://github.com/eclipse-ee4j/jakartaee-platform/issues/580

**The URL where Compatible Software Product is available for download**
https://www.payara.fish/downloads/payara-platform-community-edition/

### PR DESCRIPTION
**Pull request by oliviergoulet5**
Resolves #1592

### CODE REVIEW

**Comment by autumnfound** [comment]
We should be moving the Web profile version to the web profile products section rather than deleting it

**Review by autumnfound — CHANGES_REQUESTED** [review]
see above comment

**Comment by oliviergoulet5** [comment]
+1 from Tanja

**Comment by oliviergoulet5** [comment]
> We should be moving the Web profile version to the web profile products section rather than deleting it

As we discussed in #1592 with @autumnfound, this is Payara Server's web profile. Not to be confused with Jakarta EE 10's web profile.

**Review by ericpoirier — APPROVED** [review]

File: data/compatible_products.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ sets:
           image_width: 195
           download: 'https://www.payara.fish/downloads/payara-platform-community-edition/'
           versions:
-            - version: 6.2022.1
+            - version: 6.2022.1, Web Profile
```
Comment (autumnfound) [inline data/compatible_products.yml:80]: Isn't the full server also web profile compatible?

Comment (oliviergoulet5) [reply data/compatible_products.yml:80]: No, Payara full server being added to web profile wasn't part of the issue. 

Also check: https://docs.payara.fish/community/docs/6.2022.1/Jakarta%20EE%20Certification/6.2022.1/6.2022.1%20Web%20TCK%20Results.html

The sidebar has all the certs. The full server for web profile isn't listed.

**Review by autumnfound — APPROVED** [review]

**Pull request merged**